src/partials/contacts.html

```diff
@@ -1,38 +1,46 @@
 <section class="contact-section">
-  <h1 class="visually-hidden">Contacts</h1>
-  <div class="contacts-div">
+  <h2 class="visually-hidden">Contacts</h2>
+  <ul class="contacts-list">
+    <li class = "contacts-item">
     <p class="contacts-header">CAFE</p>
-    <p class="city">Chicago</p>
-    <span class="place">10340 Ridgeland Ave, Chicago Ridge, IL 60415, USA</span>
-    <p class="date">Monday - Friday</p>
-    <span class="time">06:00 AM - 10:00 PM</span>
-    <p class="date">Saturday - Sunday</p>
-    <span class="time">08:00 AM - 4:00 PM</span>
+    <p class="contacts-city">Chicago</p>
+    <span class="contacts-place">10340 Ridgeland Ave, Chicago Ridge, IL 60415, USA</span></li>
+    <li class = "contacts-item">
+    <p class="contacts-date">Monday - Friday</p>
+    <span class="contacts-time">06:00 AM - 10:00 PM</span>
+    <p class="contacts-date">Saturday - Sunday</p>
+    <span class="contacts-time">08:00 AM - 4:00 PM</span></li>
+    <li class = "contacts-item">
     <a href="#">+61(0) 383 766 284</a>
-    <a href="#">noreply@envato.com</a>
-  </div>
-  <div class="contacts-div">
+    <a href="#">noreply@envato.com</a></li>
+  </ul>
```

Comment:
Kamilowi chodzi o to, że co przed tym było `<div>`, a teraz jest `<ul>`, powinno być jeden `<li>`

Reply:
no ma być coś takiego 
<ul >
<li>
<li>
<li>
</ul>

Reply:
Już to robię